test/parallel/forall/vass/other/visibility-of-record-for-reduce.chpl

```diff
@@ -2,7 +2,7 @@
 module B {
 
   record foo { var v:int; }
-  proc +(a:foo,b:foo):foo { return new foo(a.v+b.v); }
+  operator foo.+(a:foo,b:foo):foo { return new foo(a.v+b.v); }
```

Comment:
I suggest making operators in this test non-methods. It will illustrate that they are visible in the current module even when they are not methods.